tests/org.dataflowanalysis.converter.tests/src/org/dataflowanalysis/converter/tests/PCMTest.java

```diff
@@ -87,68 +92,59 @@ private void testSpecificModel(String inputModel, String inputFile, String model
 
         var dfd = complete.dataFlowDiagram();
         var dd = complete.dataDictionary();
+        
 
-        assertEquals(dfd.getNodes()
-                .size(),
-                assIdToName.keySet()
-                        .size());
-
-        List<String> nodeIds = new ArrayList<>();
-        for (Node node : dfd.getNodes()) {
-            nodeIds.add(node.getId());
-        }
-        Collections.sort(nodeIds);
-        List<String> assIds = new ArrayList<>(assIdToName.keySet());
-        Collections.sort(assIds);
-
-        assertEquals(assIds, nodeIds);
-
-        for (Node node : dfd.getNodes()) {
-            assertEquals(node.getEntityName(), assIdToName.get(node.getId()));
```

Comment:
why cant we check node names anymore?

Reply:
Has to do with [this](https://github.com/DataFlowAnalysis/Converter/pull/9#discussion_r1722891434):
As multiple DFD elements are created from PCM elements already, and some names just being equal in PCM elements (like the `aName` for Start and Stop Elements), names never have been sufficient to compare elements in PCM

Reply:
Implemented in 6acdf117b40c